csi/src/meta_data.rs

```diff
@@ -993,11 +1067,12 @@ impl MetaData {
             BindMountMode::Single => {
                 let write_res = self
                     .etcd_client
-                    .write_new_kv(&volume_mount_path_key, &mount_path_value_in_etcd);
+                    .write_new_kv(&volume_mount_path_key, &mount_path_value_in_etcd)
+                    .await;
                 if let Err(e) = write_res {
                     panic!(
                         "failed to write the mount path={} of volume ID={} to etcd, \
-                            the error is: {}",
+                        the error is: {}",
```

Comment:
extra 4 spaces indentation before the second text line

Reply:
Done